strawberry/asgi/handlers/http_handler.py

```diff
@@ -54,7 +48,12 @@ async def handle(self, scope: Scope, receive: Receive, send: Send):
             context=context,
         )
 
-        response.headers.raw.extend(sub_response.headers.raw)
+        # Only add headers from sub_response if they're not already present in response
+        response.headers.raw.extend(
+            (k, v)
+            for k, v in sub_response.headers.raw
+            if k not in {k for k, _ in response.headers.raw}
+        )
```

Comment:
should we add a warning if we are trying to override some headers?

Reply:
Up to you, this is currently blocking `content-length: 0` from appearing and being a duplicate which also crashes starlette.

Reply:
let's do that if you don't mind 😊 I think it might be helpful in cases where you might set headers twice